src/main/java/ti4/buttons/handlers/agenda/AgendaResolveButtonHandler.java

```diff
@@ -1061,13 +1061,47 @@ public static void resolveAgenda(Game game, String buttonID, ButtonInteractionEv
         if (!game.isOmegaPhaseMode()) {
             buttons.add(Buttons.green("proceed_to_strategy", "Proceed to Strategy Phase (will run agenda cleanup and ping speaker)"));
         } else {
-            Button electVoiceOfTheCouncil = Buttons.green("elect_voice_of_the_council", "Elect Voice of the Council");
+            Button proceedToScoring;
+            Button electVoiceOfTheCouncil;
+            String currentVotC = game.getLawsInfo().get(Constants.VOICE_OF_THE_COUNCIL_ID);
+            boolean mustElectVoice = currentVotC == null || currentVotC.isEmpty();
+            String electVoiceText = "Elect Voice of the Council";
+            if (mustElectVoice) {
+                electVoiceText += " (required before scoring)";
+            } else {
+                electVoiceText += " (optional)";
+            }
+            if (aCount < 3) {
+                electVoiceOfTheCouncil = Buttons.gray("elect_voice_of_the_council", electVoiceText);
+                proceedToScoring = Buttons.gray("proceed_to_scoring", "Proceed to scoring objectives");
+            } else {
+                electVoiceOfTheCouncil = Buttons.green("elect_voice_of_the_council", electVoiceText);
+                if (mustElectVoice) {
+                    proceedToScoring = Buttons.gray("proceed_to_scoring", "Proceed to scoring objectives");
+                } else {
+                    String speakerUserID = game.getSpeakerUserID();
+                    Player speaker = game.getPlayer(speakerUserID);
+                    if (speaker != null) {
+                        MessageChannel speakerCardsInfoThread = speaker.getCardsInfoThread();
+                        MessageHelper.sendMessageToChannel(speakerCardsInfoThread, "These are the current votes available for the Voice of the Council vote.");
+                        AgendaHelper.listVoteCount(game, speakerCardsInfoThread);
+                    }
+                    proceedToScoring = Buttons.green("proceed_to_scoring", "Proceed to scoring objectives");
+                }
+            }
+            if (aCount == 3) {
+                voteMessage += " The bot believes this is the 3rd agenda, which in Omega Phase means you might vote on the Voice of the Council.";
+                voteMessage += "\n- If a player currently has the Voice of the Council, the Speaker chooses whether to vote on it this round or not.";
+                voteMessage += "\n- If no player currently has the Voice of the Council, it must be voted on once before proceeding to scoring.";
+                voteMessage += "\nIf this is not actually the third agenda yet, please remember this when that agenda is reached.";
+            }
```

Comment:
Print this helper text for the 3rd agenda to help players figure out which button to use

Reply:
Is `aCount` a little fuzzy? It seems like `"miscount"` will cause it to increase, but it is kept accurate for Quash and Veto. If we're very confident that `aCount` is accurate, then this blurb should probably be replaced with smarter button colors/text.